### ISSUE TITLE
request for `kpt live get` 

### ISSUE THREAD

**Issue by Liujingfang1** [GoogleContainerTools/kpt#303]
Similar to `kubectl get`, we can have a `kpt live get` command to get the status of resources in a package.

**Comment by Liujingfang1** [comment]
btw, `kubectl get` doesn't work with a kpt package with a grouping object.

**Comment by mikebz** [comment]
@Liujingfang1 I am not sure I follow this exactly.  You'd like to get all of the resources in the package?

**Comment by Liujingfang1** [comment]
Yes, with this command, users can see the resources applied to cluster and their status. It might be similar to the `kpt live status`. In that case, we can close this issue.
The more interesting case is for targeting multiple cluster or filtering out certain resources that don't target the current cluster. This is a different problem, we can track it as a different issue.
